Frameworks/CoreText/CTFont.mm

```diff
@@ -493,17 +493,16 @@ CFStringRef CTFontCopyDisplayName(CTFontRef font) {
  @Status Interoperable
 */
 CFStringRef CTFontCopyName(CTFontRef font, CFStringRef nameKey) {
-    RETURN_NULL_IF(!font);
-    return _DWriteFontCopyName(font->_dwriteFontFace, nameKey);
+    return CTFontCopyLocalizedName(font, nameKey, nullptr);
 }
 
 /**
- @Status Stub
+ @Status Interoperable
  @Notes
 */
 CFStringRef CTFontCopyLocalizedName(CTFontRef font, CFStringRef nameKey, CFStringRef _Nullable* actualLanguage) {
-    UNIMPLEMENTED();
-    return StubReturn();
+    RETURN_NULL_IF(!font);
```

Comment:
why is this a `RETURN_NULL_IF` but the other functions in this file are `return x ? : nullptr`?